validators/performance/aiperf-bench.Dockerfile

```
@@ -83,8 +89,27 @@ COPY validators/performance/requirements.txt /tmp/requirements.txt
+#      with no legitimate runtime caller.
+#   2. Open-source compliance. pip is third-party software this layer adds on
+#      top of the approved base -- the base carries no Python distributions, so
+#      pip is ours, not inherited -- and shipping it obliges us to publish its
+#      source. It is also the one package whose version floats independently of
+#      requirements.txt, via the --upgrade above, so that source would have to
+#      track whatever pip happened to be newest at build time.
+#
+# Nothing in the installed closure declares pip as a dependency, so removing it
+# cannot break a runtime import. Verified by building both ways: `aiperf
+# --version` and aiperf_entrypoint.py behave identically with and without it.
 RUN pip install --no-cache-dir --upgrade pip \
```

Hadolint (2.15.1)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)